rfcs/0000-address-format/0000-address-format.md

```diff
@@ -0,0 +1,49 @@
+---
+Number: 0000
+Category: <TBD>
+Status: Proposal
+Author: Cipher Wang
+Organization: Cryptape Ltd
+Created: 2019-01-20
+---
+
+# Address Format
+
+## Abstract
+
+CKB Address Format is an application level [lock script][script-define] display recommendation. The lock script consists of three key parameters, including version, args, and binary_hash. In the consideration of user experience, it is necessary to wrap the raw data structure into a verifiable and extensible format.
+
+## Solution
+
+CKB Address Format follows [Bitcoin base32 address format (BIP-173)][bip173] rules, which add a version prefix to the payload (such as binary_hash and private key), and wrap them in **Bech32** encoding and a [BCH checksum][bch].
+
+A Bech32 string is at most 90 characters long and consists of the **human-readable part**, the **separator**, and the **data part**.
+
+The human-readable part is "ckb" for CKB mainnet, and "cbt" for the testnet. The separator is always "1". The data part is consist of version prefix and payload. The version part means the address format version, and currently fixed to 0x00. The payload part encodes lock script by three data fields, payload = type | binary_ref | args.
```

Comment:
`cbt` should be `ckb` according to our previous discussions.

Reply:
You mean ‘ckt’ or ‘cbt’?

Reply:
ckt, np.